controller/controller_server/messages.py

```diff
@@ -3,7 +3,12 @@
 ObjectIdError_message = "Wrong {0} id format : {1}"
 
 # validation error messages
-invalid_secret_message = "invalid secret was passed"
+invalid_secrets_config_message = "invalid secrets config was passed"
+no_way_to_identify_system_from_secrets_config_message = "no way to identify system from secrets config"
```

Comment:
I wouldn't call it `identify`, perhaps something like `insufficient data to choose a storage system`?

Reply:
done